src/stylesheets/button.scss

```diff
@@ -58,6 +58,7 @@ button.btn, a.btn {
   }
   &.centered {
     margin: auto;
+    font-size: 1.2vw;
```

Comment:
Why `vw`?

Reply:
So that the font size automatically adjusts to fit inside the button.  Otherwise--as you pointed out elsewhere in this PR--if your button text is too long, it goes off the edge of the button and gets cut off.